datumaro/components/launcher.py

```diff
@@ -18,6 +18,9 @@ def __init__(self, model_dir=None):
     def launch(self, inputs):
         raise NotImplementedError()
 
+    def infos(self):
+        return None
+
```

Comment:
Do we have to remove these lines?

Reply:
Is there some reason to remove it? I added these lines because 'categories' did a similar thing in the following lines.

Reply:
We have two duplicated `infos(self)` functions for now.

Reply:
In my best understanding, class Launcher has one infos(self) function by these lines. If I miss something, please kindly point the duplicated functions.

Reply:
It was my mistake. There actually are `Launcher.infos()` and `ModelTransform.infos()`. I'll resolve this thread.